index.html

```diff
@@ -207,6 +212,35 @@ <h3>Specific Verifiable Credentials</h3>
 that should work for most use cases.
         </p>
       </section>
+      <section>
+        <h3>Content Integrity</h3>
+        <p>
+Protecting the integrity of content is an important component of verification.
+<a>Verifiers</a> need to have confidence that the content they rely on to
+verify <a>credentials</a> doesn't change without their knowledge. This content
+may include data schemas, identifiers, public keys, etc.
+        </p>
+        <p>
+There are a number of ways to provide content integrity protection. A few of
+them are provided in greater detail below.
```

Comment:
`them are provided`
-> `these are described`

Reply:
fixed in [058b883](https://github.com/w3c/vc-imp-guide/pull/40/commits/058b883d0adafa4a54c8c49fddce42146f84ba31)

Reply:
`them` remains ... but I'll live with it

Reply:
and [9faa5c7](https://github.com/w3c/vc-imp-guide/pull/40/commits/9faa5c74747744aac2da8c1c9ab646132ba69118)